README.md

```diff
@@ -375,6 +375,14 @@ thus avoiding costly network transfers for shared directories.
 
 * **mount_path** : the path to replace with a tmpfs
 
+### Mailcatcher
+
+With mailcatcher it's very easy to test an application which sends emails. It allows you to catch emails right in the box and they weren't redirected to their real destination (no more spamed dev inbox). 
```

Comment:
I still don't think this is english ;)
"right in the box instead of them being sent to their real destination" will sound better I think.